tests/suites/test_suite_pkcs12.function

```diff
@@ -15,7 +17,7 @@ typedef enum
  * END_DEPENDENCIES
  */
 
-/* BEGIN_CASE */
+/* BEGIN_CASE depends_on:MBEDTLS_HAS_ALG_MD5_VIA_MD_OR_PSA */
```

Comment:
I'm not a big fan of this. IMO dependency declarations in `.function` files should be for compile-time dependencies, not just a shortcut for "all the test cases happen to require this". In the future we might want to add test cases using other algorithms than MD5 (for example if a bug is found that only occurs with larger hashes, or if at some point we completely remove MD5 but still want to support and test PKCS12).

So, unless there's a reason I missed for having this here, I'd prefer to keep things as they were, with each test case declaring its own dependency (just changing to the new macro, obviously).

Reply:
I valued readability and less clutter in favour of possible future expansion of tests - the requirement could then be changed and there's no way to miss it. Fixed according to your suggestions though :)

Reply:
Thanks!